Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM continuumio/miniconda3:4.9.2
+
+RUN mkdir -p /usr/local/src
+WORKDIR /usr/local/src
+
+COPY requirements-conda.dev.txt /usr/local/src/requirements-conda.dev.txt
```

Comment:
I'd consider renaming this file `requirements-conda-dev.txt`.

Reply:
Done.